database.rules.json

```diff
@@ -26,6 +26,10 @@
       ".read": false,
       ".write": true
     },
+    "potentialPartners": {
+      ".read": false,
+      ".write": true
```

Comment:
it's possible to erase all data by anyone

Reply:
Anyone can write there. What you can propose?

Reply:
you can disallow writing to the root of `potentialPartners`, but allow write to it's childs

Reply:
please give an example, and I'll change at other places